src/js/media-container.js

```diff
@@ -359,13 +374,15 @@ class MediaContainer extends window.HTMLElement {
 
     // Immediately hide if mouse leaves the container
     this.addEventListener('mouseleave', (e) => {
-      if (this.autohide < 0) return;
-      this.setAttribute('user-inactive', 'user-inactive');
-      const evt = new window.CustomEvent(
-        MediaStateChangeEvents.USER_INACTIVE, 
-        { composed: true, bubbles: true, detail: true }
-      );
-      this.dispatchEvent(evt);
+      setInactive();
+    });
+
+    // Allow for focus styles only when using the keyboard to navigate
+    this.addEventListener('keyup', (e) => {
+      this.setAttribute('media-keyboard-control', 'media-keyboard-control');
```

Comment:
```suggestion
      this.setAttribute('media-keyboard-control', '');
```

for attr booleans this is the platform way

Reply:
Sounds good. As an FYI, this is how it was previously.